File: Last_3_Number_Reverse.c
#include<stdio.h>
void main(){
    int num=123456;
    int r;
    int f= num/1000;
    while(num>999){
        printf("%d\n", num);
        r = num%10;
        f = f*10+r;
        num = num/10;
    }
    // printf("%d",f);
}

File: Last_3_Number_Reverse.js
let num = 123456;
let f = Math.floor(num/1000);
let remainder=0;
console.log("Orignal number is: " + num);
while(num>999){
    // console.log(num);
    remainder = num % 10;
    f = f * 10 + remainder;
    num = Math.floor(num/10);
}
console.log("Reverse number is: " + f);